core/build.gradle

```diff
@@ -231,7 +228,7 @@ dependencies {
 
   implementation group: 'com.cloudera.cdp',                   name: 'cdp-sdk-java'
 
-  implementation (group: 'com.nimbusds',                      name: 'oauth2-oidc-sdk',                  version: '6.5') {
+  implementation (group: 'com.nimbusds',                      name: 'oauth2-oidc-sdk',                  version: '9.20') {
```

Comment:
did you try to start Azure cluster? It was fixed to 6.5 because we got NoClassDefFound when we started Azure clusteres